src/components/banner/_banner.scss

```diff
@@ -73,51 +71,50 @@ $logo-breakpoint: 1400px;
   }
 
   &__logo {
-    width: 3.3em;
+    width: $size-6;
 
-    @media screen and (min-width: 90em) {
-      left: 2em;
-      top: 3em;
+    @media screen and (min-width: map-get($breakpoints, "breakpoint-10")) {
+      left: $spacing-6;
+      top: $spacing-7;
     }
 
-    @media screen and (min-width: $logo-breakpoint) {
+    @media screen and (min-width: map-get($breakpoints, "breakpoint-10")) {
       position: fixed;
       z-index: 1000;
 
       svg {
         opacity: 0;
         transform: translateY(50%);
-        transition:
-          opacity 400ms ease,
-          transform 400ms ease;
+        transition: opacity $duration-5 $timing-ease,
+          transform $duration-5 $timing-ease;
       }
 
       &--visible {
         svg {
           opacity: 1;
-          transform: translateY(0)
+          transform: translateY(0);
         }
       }
     }
   }
 
   &__button {
     background: rgba(53, 54, 61, .3);
-    border: 0;
-    border-radius: 50%;
+    border: none;
+    border-radius: $border-radius-half;
     cursor: pointer;
-    height: 2em;
+    height: $size-3;
     padding: 0;
     position: relative;
-    transition: background .2s ease-in-out;
-    width: 2em;
+    transition: background $duration-2 $timing-ease-in-out;
+    width: $size-3;
     z-index: 1;
     -webkit-appearance: none;
```

Comment:
`-webkit-appearance: none;` should be treated as an **a** in the alphabet; not **-** or **w**.